modules/users/client/views/admin/view-user.client.view.html

```diff
@@ -8,7 +8,7 @@ <h1 ng-bind="vm.user.username"></h1>
         <a class="btn btn-primary" ui-sref="admin.user-edit({userId: vm.user._id})">
           <i class="glyphicon glyphicon-edit"></i>
         </a>
-        <a class="btn btn-primary" ng-click="vm.remove()" ng-if="vm.user._id !== vm.authentication.user._id">
+        <a class="btn btn-primary" ng-click="vm.remove()" ng-if="vm.user.username !== vm.authentication.user.username">
```

Comment:
Using `username` here is OK. It always feels weird to me though, to compare using a field like this, vs the actual `_id` or some other more intentional field.

Instead of doing the comparison on this field in the template, how about we create a flag that we manage in the controller using `vm`. Something like: `vm.isCurrentUserSelf = isSelf();` in the controller. By doing it this way, we can avoid the need to update the template if we ever change how we're determined what "self" means in this context.